packages/react/src/components/Menu/MenuItem.tsx

```diff
@@ -57,6 +57,12 @@ export interface MenuItemProps
   /** MenuItem index inside Menu. */
   index?: number
 
+  /** MenuItem position inside Menu (skipping separators). */
+  itemPosition?: number
+
+  /** MenuItem count inside Menu. */
+  itemsCount?: number
```

Comment:
These props will be added to public API actually, I'm fine with this for now, but I'm not sure that components should be so coupled with accessibility concerns.

Reply:
yes, I am aware. but as far as I know, we do not have any other option to pass the information from parent menu to the menu item behaviors.